backend/src/Manager/Notification/NotificationFirebaseManager.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+
+namespace App\Manager\Notification;
+
+use App\AutoMapping;
+use App\Entity\NotificationFirebaseTokenEntity;
+use App\Repository\NotificationFirebaseTokenEntityRepository;
+use App\Request\Notification\NotificationFirebaseTokenCreateRequest;
+use Doctrine\ORM\EntityManagerInterface;
+use App\Manager\User\UserManager;
+
+class NotificationFirebaseManager
```

Comment:
if this class is for  notifications tokens lets us change its name to something like "NotificationTokens"